packages/backend/src/modules/tvl/TvlModule.ts

```diff
@@ -40,113 +29,81 @@ export function createTvlModule(
   database: Database,
   clock: Clock,
 ): ApplicationModule | undefined {
-  if (!config.tvl) {
+  if (!config.tvl.enabled) {
     logger.info('TVL module disabled')
     return
   }
-
   // #region database
-  const blockNumberRepository = new BlockNumberRepository(database, logger)
-  const priceRepository = new PriceRepository(database, logger)
-  const balanceRepository = new BalanceRepository(database, logger)
-  const reportRepository = new ReportRepository(database, logger)
-  const aggregatedReportRepository = new AggregatedReportRepository(
-    database,
-    logger,
-  )
-  const reportStatusRepository = new ReportStatusRepository(database, logger)
-  const balanceStatusRepository = new BalanceStatusRepository(database, logger)
 
+  const db = {
+    blockNumberRepository: new BlockNumberRepository(database, logger),
+    priceRepository: new PriceRepository(database, logger),
+    balanceRepository: new BalanceRepository(database, logger),
+    reportRepository: new ReportRepository(database, logger),
+    aggregatedReportRepository: new AggregatedReportRepository(
+      database,
+      logger,
+    ),
+    reportStatusRepository: new ReportStatusRepository(database, logger),
+    balanceStatusRepository: new BalanceStatusRepository(database, logger),
+  }
   // #endregion
+
   // #region peripherals
 
-  const ethereumProvider = new providers.AlchemyProvider(
-    'mainnet',
-    config.tvl.alchemyApiKey,
-  )
-  const ethereumClient = new EthereumClient(ethereumProvider, logger)
-  const multicall = new MulticallClient(ethereumClient)
   const coingeckoClient = new CoingeckoClient(http, config.tvl.coingeckoApiKey)
   const coingeckoQueryService = new CoingeckoQueryService(coingeckoClient)
-  const etherscanClient = new EtherscanClient(
-    http,
-    config.tvl.etherscanApiKey,
-    logger,
-  )
-  const ethereumBalanceProvider = new EthereumBalanceProvider(multicall)
 
   // #endregion
+
   // #region updaters
 
-  const blockNumberUpdater = new BlockNumberUpdater(
-    etherscanClient,
-    blockNumberRepository,
-    clock,
-    logger,
-    ChainId.ETHEREUM,
-  )
   const priceUpdater = new PriceUpdater(
     coingeckoQueryService,
-    priceRepository,
+    db.priceRepository,
     clock,
     config.tokens,
     logger,
   )
-  const balanceUpdater = new BalanceUpdater(
-    ethereumBalanceProvider,
-    blockNumberUpdater,
-    balanceRepository,
-    balanceStatusRepository,
-    clock,
-    config.projects,
-    logger,
-    ChainId.ETHEREUM,
-  )
-  const reportUpdater = new ReportUpdater(
-    priceUpdater,
-    balanceUpdater,
-    reportRepository,
-    aggregatedReportRepository,
-    reportStatusRepository,
-    clock,
-    config.projects,
-    logger,
-  )
 
   // #endregion
+
   // #region api
```

Comment:
What's the deal with those extra spaces? 
```suggestion
  // #endregion
  // #region api
```